Sources/Testing/Support/CError.swift

```diff
@@ -8,7 +8,7 @@
 // See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 //
 
-private import TestingInternals
+internal import TestingInternals
```

Comment:
Should we remove the `internal`? There are only 2 other hits for `internal import`, all other usages are implicit.

Reply:
It's necessary because this file uses a type from TestingInternals in an `internal` spot, and because other spots say `private`, we need to explicitly state that this import is `internal`.